app/views/state_file/questions/md_permanently_disabled/edit.html.erb

```diff
@@ -11,20 +11,18 @@
     <div class="question-with-follow-up">
       <div class="question-with-follow-up__question">
         <div class="white-group">
+          <% could_qualify_for_pension_exclusion = current_intake.could_qualify_for_pension_exclusion? %>
```

Comment:
would we show this page if there were no eligible 1099Rs?  (that is 1099Rs with a taxable amount) because currently the logic shows this page if `intake.state_file1099_rs.length.positive?`

Reply:
Yes that is a requirement of the story.

Reply:
can confirm that we need to show this page even if 1099Rs all do not have taxable amt -- we need to show this page if we're including the MD502R form in the xml bundle, which happens under a different set of logic